### PR TITLE
Add support for VS2017

### PR DESCRIPTION
**Pull request by jpozo20**
Added Support for VS2017 by linking to the source files in the VS2015 and using the correct assemblies for VS2017

Also reorganized the projects so they are at the same level, differing by the version it supports.

### CODE REVIEW

File: ExportExtensios2017/app.config [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                </binding>
+                <binding name="CustomBinding_IVsIdeService1">
+                    <security mode="Transport">
+                        <transport clientCredentialType="None" />
+                    </security>
+                </binding>
+            </wsHttpBinding>
+        </bindings>
+        <client>
+            <endpoint address="http://co1msdngal97.phx.gbl/Services/dev12/Extension.svc"
```
Comment (Mike-E-angelo) [inline ExportExtensios2017/app.config:17]: Thanks for your contribution. :)  Looks like this is a local URL... is that intentional?

Comment (jpozo20) [reply ExportExtensios2017/app.config:17]: I don't remember well, I think it's something about the extension. I'll be checking it out today.

EDIT:
After doing a quick google search, It looks like that url corresponds to this one:
https://visualstudiogallery.msdn.microsoft.com/Services/VStudio/Extension.svc?wsdl

But I don't know why it got encoded like that lol, I'll be changing and committing it in a few minutes 👍🏼

**Comment by laedit** [comment]
Thanks for this update to VS2017 😃 
Any news on when it will be completed?

File: ExportExtensios2017/app.config (hunk 1 above)
Comment (stefnotch) [reply ExportExtensios2017/app.config:17]: @jpozo20 @Mike-EEE Any updates regarding this extension?

**Comment by hrai** [comment]
I'm still having issues with this extension on vs2017. The menu item doesn't even show on the menu.

**Pull request merged**